### ISSUE TITLE
Before GO-LIVE: Hierarchy Search: Ability to select/add from different Level for PT and LLT levels.

### ISSUE THREAD

**Issue by zmeenakshi** [DBMS-Consulting/CQT2#361]
Module : All 
Page: Hierarchy Search in Relations and Impact Assessment

Requirement: The user should not be CONFINED to the original level selected in the hierarchy search after drilling/navigating to different level. ** If they navigate to a different level, the user should be able to add that dictionary term from there itself**

Issue: When user search for SOC, HLGT and HLT level terms, they can select and add  terms from other levels while drilling down the hierarchy, but this is NOT possible when search for terms at PT and LLT levels. 
The entire term hierarchy gets selected when trying to select any term from HS under PT and LLT:

![image](https://cloud.githubusercontent.com/assets/25993316/26451201/a144c156-4178-11e7-98ab-c43df779e631.png)

Works as expected for SMQs, SOC, HLGT, HLT:
![image](https://cloud.githubusercontent.com/assets/25993316/26451255/e73de2c8-4178-11e7-94b2-770ef2772281.png)

**Comment by zmeenakshi** [comment]
Any update on this one?

**Comment by andmiel81** [comment]
I had to change the selection mode of hierarchy search table.
Primefaces provides 3 selection mode for tree table: single, multiple, and checkbox.
Here, modes that supports multiple select are multiple and checkbox.

- Originally the mode was checkbox, with the checkboxes hidden.
In this mode, when all the children nodes are selected, the parent node is automatically selected.
And the selection can be toggled by simply clicking on the node, once and twice.

- Now the mode has been changed to multiple.
In this mode, we can choose multiple nodes by clicking on nodes while pressing CTRL key.
Also in this mode, even though all the children nodes are selected, the parent node won't be automatically selected.
Problem solved, but user action for multi-selection has been changed.
